test/lib/course_planner/accounts/user_test.exs

```diff
@@ -105,4 +137,18 @@ defmodule CoursePlanner.UserTest do
       assert changeset.errors[:notification_period_days] == {"must be less than or equal to %{number}", [validation: :number, number: 7]}
     end
   end
+
+  describe "password changeset" do
+    test "when setting password" do
+      coordinator = insert(:coordinator)
+      changeset = User.changeset(coordinator, %{password: "secret", password_confirmation: "secret"}, :password)
+      assert [] == changeset.errors
```

Comment:
Is there a reason for not using `assert changese.valid?`?

Reply:
no there is no reason, i think i was just lost

Reply:
done